lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,53 @@
+module MigrationSyncHelper
```

Comment:
Should this module live *inside* the ArPglogicalMigration namespace, so it's clear it's not for external use?

Reply:
It could. My intention here was to avoid mixing in more than just the `#migrate` method to `ActiveRecord::Migration`, but if we nest this module in `ArPglogicalMigration` then it seems okay. I'll probably rename it to `ArPglogicalMigrationHelper` though.